src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
From what I have learned recently, some "ANSI" APIs actually internally interpret the buffer as UTF8 encoded. CoreCLR hosting APIs are this way, for example.